config/configvalidator.php

```diff
@@ -32,8 +32,8 @@ public function isConfigSafe($key, $parsedConfigItem) {
 
 		switch ($key) {
 			case 'sorting':
-				$safe = $this->isSortingTypeSafe($parsedConfigItem, $safe);
-				$safe = $this->isSortingOrderSafe($parsedConfigItem, $safe);
+				$safe = $this->isSortingSafe( 'type',$parsedConfigItem, $safe);
```

Comment:
spacing.
Make sure to ask your IDE to format the text before making the commit.